### PR TITLE
feat: [KDL6-206] add labels to secrets

### PR DESCRIPTION
**Pull request by igzjaviergil**
## Motivation and Context

Add label konstellation.io/ssh-key: true to <user>-ssh-keys 
Add label konstellation.io/secret-service-account-token: true to <user>-service-account-secret
Add tests

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] I have created tests for my code changes, and the tests are passing.
- [X] I have executed the pre-commit hooks locally.
- [ ] I have updated the documentation accordingly.
- [X] The commit message follows our guidelines: https://github.com/konstellation-io/kdl-server/blob/main/CONTRIBUTING.md
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: app/api/http/main.go [hunk 1]
```diff
@@ -123,7 +124,12 @@ func startHTTPServer(
 ) {
 	const apiQueryPath = "/api/query"
 
-	srv := handler.NewDefaultServer(generated.NewExecutableSchema(generated.Config{Resolvers: resolvers}))
+	srv := handler.New(generated.NewExecutableSchema(generated.Config{Resolvers: resolvers}))
+	srv.AddTransport(transport.Websocket{
+		KeepAlivePingInterval: 30 * time.Second,
+	})
+	srv.AddTransport(transport.POST{})
```
Comment (alexrodfe) [inline app/api/http/main.go:131]: I'd add here as well:

```
	srv.AddTransport(transport.GET{})
	srv.AddTransport(transport.POST{})
	srv.AddTransport(transport.MultipartForm{})
	``

File: app/api/infrastructure/k8s/config.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
+package k8s
```
Comment (alexrodfe) [inline app/api/infrastructure/k8s/config.go:1]: for starters this file should end with '_test.go' so, a its proper name should be: 'config_test.go'

Comment (alexrodfe) [reply app/api/infrastructure/k8s/config.go:1]: An the first line should include the "go build tag integration"

Comment (alexrodfe) [reply app/api/infrastructure/k8s/config.go:1]: also package should be `k8s_test`

File: app/api/infrastructure/k8s/user_ssh.go [hunk 3]
```diff
@@ -18,26 +18,29 @@ func (k Client) getUserSSHSecretName(usernameSlug string) string {
...
 
 // newUserSSHSecret returns the name and the k8s secret for public and private SSH keys.
-func (k *Client) newUserSSHSecret(user entity.User, public, private string) (secretName string, secretValues map[string]string) {
+func (k *Client) newUserSSHSecret(user entity.User, public, private string) (secretName string, secretValues, labels map[string]string) {
 	secretName = k.getUserSSHSecretName(user.UsernameSlug())
 	secretValues = map[string]string{
 		KdlUserPublicSSHKey:  public,
 		KdlUserPrivateSSHKey: private,
 	}
+	labels = map[string]string{
```
Comment (alexrodfe) [inline app/api/infrastructure/k8s/user_ssh.go:27]: watchout, you are overwritting "labels" value here

**Review by alexrodfe — CHANGES_REQUESTED** [review]

File: app/api/http/main.go (hunk 1 above)
Comment (igzjaviergil) [reply app/api/http/main.go:131]: I wanted to be more restrictive, but indeed it was like this before so keeping it

File: app/api/infrastructure/k8s/config.go (hunk 2 above)
Comment (igzjaviergil) [reply app/api/infrastructure/k8s/config.go:1]: noted!

File: app/api/infrastructure/k8s/user_ssh.go (hunk 3 above)
Comment (igzjaviergil) [reply app/api/infrastructure/k8s/user_ssh.go:27]: well spotted, I just copied pasted and didn´t realized

**Review by alexrodfe — DISMISSED** [review]

**Review by alexrodfe — APPROVED** [review]

**Pull request merged**